lib/to-lua.nix

```diff
@@ -15,12 +15,17 @@ rec {
         + (concatStringsSep "," (
           mapAttrsToList (
             n: v:
-            if (builtins.match "__unkeyed.*" n) != null then
-              toLuaObject v
+            let
+              valueString = toLuaObject v;
```

Comment:
```suggestion
              lua = toLuaObject v;
```
What do you think?

Reply:
I think that `valueString` is clearer, but again, no strong opinion X)

Reply:
I think `lua` implies the string has already been rendered/formatted as lua, but I also don't have a strong opinion.

`luaValue` and `luaString` could also work, though `lua` has the advantage of also being less verbose.